core/slice.go

```diff
@@ -62,17 +62,22 @@ type Slice struct {
 	bestPhKey common.Hash
 	phCache   map[common.Hash]types.PendingHeader
 
-	validator Validator // Block and state validator interface
+	validator  Validator // Block and state validator interface
+	tpsHistory *lru.Cache
+	subTps     []uint32
```

Comment:
Not sure I like relaying subordinate TPS measurements up to be aggregated. The zones are already submitting their measurements. The stats backend can perform any aggregation there. Anything we don't have to compute in the node, we shouldn't.